test/acceptance/integration/notifications/hadoop/hadoop_row_multi_instances.feature

```diff
@@ -31,9 +31,9 @@ Feature: start multi-instances of cygnus using hadoop sink and row mode
 
   @happy_path @multi_instances
   Scenario Outline: start multi-instances of cygnus using hadoop sink, row mode, ports differents and store multiples notifications one by instance and the port defined incremented
-    Given copy properties.json file from "epg_properties.json" to test "hdfs-sink" and sudo local "false"
+    Given copy properties.json file from "filab_properties.json" to test "hdfs-sink" and sudo local "false"
```

Comment:
Regarding the file replacing, it could be interesting to make it before tests execution in a Jenkins task or shell script, otherwise you need to change the feature every time you change the environment.